### ISSUE TITLE
(fabric 1.19) OverpoweredMending Compatibility Issue

### ISSUE THREAD

**Issue by Roboboy5741** [Lizard-Of-Oz/Inventorio#148]
OverpoweredMending is a mod that allows exp to mend any tool or piece of armor regardless of if being worn or held as long as it's in the inventory it gets repaired.

Issue:
Tools on the tool belt do not get repaired and it's the same for tools in the extra inventory that comes with the deep pockets enchant.

**Comment by Lizard-Of-Oz** [comment]
Yeah, that'd need to manually make one of mods know about each other.
Since Inventorio 2.0 is in development, I don't see much point in making that happen.

**Comment by Roboboy5741** [comment]
Will inven 2.0 come with that compatibility?

**Comment by Lizard-Of-Oz** [comment]
Hope so. Depends if I'll be able to maintain it long-term, but _my hopes_ is to have as much mod compatibility as possible, at least for the essential mods.

You can read more here: https://github.com/Lizard-Of-Oz/Inventorio/issues/126

**Comment by Roboboy5741** [comment]
Well thanks and good luck to the development hope to see its release soon i
know ill be getting it :P (byw will smite 5 axes be prioritized over sharp
5 sword when attacking pigmen, zombies ect.?)

**Comment by Lizard-Of-Oz** [comment]
> will smite 5 axes be prioritized over sharp 5 sword when attacking pigmen, zombies ect.?
Noted.

**Comment by Roboboy5741** [comment]
Thank you very much 😊

I know cosing is stressful and can be a headache so ill leave you to it.

Remember to stay hydrated/fed and get sleep also regularly get up and move
because blood clots could form in your legs if you sit for too long and
that could be dangerous.

Kind Regards
LtRobo
